src/lib/identity/api.ts

```diff
@@ -118,15 +118,20 @@ const isUserLoggedIn = (): boolean => getUserFromCookie() !== null;
 
 const getAuthStatus = async (): Promise<AuthStatus> => {
 	if (useOkta) {
-		const { isSignedInWithOktaAuthState } = await import('./okta');
-		const authState = await isSignedInWithOktaAuthState();
-		if (authState.isAuthenticated) {
-			return {
-				kind: 'SignedInWithOkta',
-				accessToken: authState.accessToken,
-				idToken: authState.idToken,
-			};
+		try {
+			const { isSignedInWithOktaAuthState } = await import('./okta');
+			const authState = await isSignedInWithOktaAuthState();
+			if (authState.isAuthenticated) {
+				return {
+					kind: 'SignedInWithOkta',
+					accessToken: authState.accessToken,
+					idToken: authState.idToken,
+				};
+			}
+		} catch (e) {
+			console.error(e);
```

Comment:
Non-blocking. Would it be useful to copy the reasons from the chat thread?

```suggestion
			// Errors:
			// the user's token is invalid/expired
			// the request to refresh the token timed out
			// the user's clock skew is too large (> 5 minutes)
			console.error(e);
```

Reply:
Just missed the merge. No worries we can add. Thanks!

Reply:
Oh that's a good suggestion, sorry I missed it! it also looks like this change has not addressed the original issue